src/components/views/elements/RoomTopic.tsx

```diff
@@ -92,7 +96,9 @@ export default function RoomTopic({
         dir="auto"
         className={className}
     >
-        { topic }
```

Comment:
I wonder why this area uses `useHover` instead of `TooltipTarget`

Reply:
I wasn't aware of `TooltipTarget` at all.